server/routes/siteMetadata.js

```diff
@@ -1,9 +1,12 @@
 import { Router } from 'express'
 import SiteMetadataController from '../controllers/siteMetadataController'
 import { v1Routes } from '../utils/routes'
+import { ensureApiAuthenticated } from '../utils/passport/ensure-api-authenticated'
 
 const router = Router()
 
-router.route(v1Routes.siteMetadata.index).post(SiteMetadataController.create)
+router
+  .route(v1Routes.siteMetadata.index)
```

Comment:
should this be an API route?

Reply:
[e8298f6](https://github.com/AMP-SCZ/dpdash/pull/440/commits/e8298f66e6af2e7b832152aab4bfbe381f3231a6)